RetailCoder.VBE/UI/Command/ExportAllCommand.cs

```diff
@@ -0,0 +1,77 @@
+using System.IO;
+using System.Windows.Forms;
+using NLog;
+using Rubberduck.VBEditor.SafeComWrappers.Abstract;
+using Rubberduck.Navigation.CodeExplorer;
+using Rubberduck.UI.CodeExplorer.Commands;
+
+namespace Rubberduck.UI.Command
+{
+    [CodeExplorerCommand]
+    public class ExportAllCommand : CommandBase 
+    {
+        private readonly IVBE _vbe;
+        private IFolderBrowserFactory _factory;
+
+        public ExportAllCommand(IVBE vbe, IFolderBrowserFactory folderBrowserFactory) : base(LogManager.GetCurrentClassLogger())
+        {
+            _vbe = vbe;
+            _factory = folderBrowserFactory;
+        }
+
+        protected override bool EvaluateCanExecute(object parameter)
+        {
+            if (parameter == null)
+            {
+                return Evaluate(_vbe.ActiveVBProject);
+            }
+            else if (parameter is CodeExplorerProjectViewModel)
+            {
+                var node = (CodeExplorerProjectViewModel)parameter;
+                return Evaluate((IVBProject)node.Declaration.Project);
```

Comment:
@retailcoder In you first example, are the `projectNode` and `project` variables created as copied objects in memory or as a new pointer to the same object in memory?

I take it that this method is more efficient, or better syntactically?